modules/nativescript/nativescript.cpp

```diff
@@ -407,6 +412,24 @@ Variant NativeScript::_new(const Variant **p_args, int p_argcount, Variant::Call
 		return Variant();
 	}
 
+	call("_init", p_args, p_argcount, r_error);
+
+	if (r_error.error != Variant::CallError::CALL_OK) {
+		instance->script = Ref<NativeScript>();
+		instance->owner->set_script_instance(NULL);
+
+#ifndef NO_THREADS
+		owners_lock->lock();
+#endif
+		instance_owners.erase(owner);
+
+#ifndef NO_THREADS
+		owners_lock->unlock();
+#endif
+
+		ERR_FAIL_COND_V(r_error.error != Variant::CallError::CALL_OK, Variant());
```

Comment:
That message would not be particularly useful though. ERR_EXPLAIN + ERR_FAIL_V would be more helpful.